src/server/frontend_wayland/xdg_activation_v1.cpp

```diff
@@ -329,21 +364,32 @@ void mf::XdgActivationV1::Instance::activate(std::string const& token, struct wl
     // 2. The surface failed to use the token in the alotted period of time
     // 3. A key was pressed down between the issuing of the token and the activation
     //    of the surface with that token
-    auto xdg_token = xdg_activation_v1->try_consume_token(token);
-    if (!xdg_token)
+    //
+
+    auto token = xdg_activation_v1->token_authority->get_token_for_string(string_token);
+    if(!token)
     {
-        mir::log_error("XdgActivationV1::activate invalid token: %s", token.c_str());
+        mir::log_error("XdgActivationV1::activate invalid token: %s", string_token.c_str());
```

Comment:
I'm not sure this is an error? Possibly a warning?

But I fear spamming the logs